engines/sci/graphics/view.cpp

```diff
@@ -927,8 +933,14 @@ void GfxView::drawScaled(const Common::Rect &rect, const Common::Rect &clipRect,
 				if (g_sci->_gfxRemap16 && g_sci->_gfxRemap16->isRemapped(outputColor))
 					outputColor = g_sci->_gfxRemap16->remapColor(outputColor, _screen->getVisual(x2, y2));
 				// SCI11+ remapping (Catdate)
-				if ((scaleSignal & 0x200) && g_sci->_gfxRemap16)
-					outputColor = g_sci->_gfxRemap16->remapColor(253, outputColor);
+				if ((scaleSignal & 0xFF00) && _resMan->testResource(ResourceId(kResourceTypeVocab, 184))) {
```

Comment:
Same here, please, readd the safeguard for g_sci->_gfxRemap16. Also, perhaps this code could be moved into a common function, to avoid duplication?